test/data/doc/dummy_doc.yaml.dt

```diff
@@ -4,7 +4,6 @@
 <location><page_1><loc_59><loc_0><loc_91><loc_75></location>
 <caption>Figure 1: Four examples of complex page layouts across different document categories</caption>
 </figure>
-<section_header><location><page_1><loc_42><loc_57><loc_49><loc_61></location>OPERATION (cont.)</section_header>
```

Comment:
Do we understand why a section header is missing?

Reply:
This is correct, because it was not supposed to be printed, since it is enclosed in a picture.